frontend/pages/login.vue

```diff
@@ -145,18 +145,28 @@ export default defineComponent({
 
     const { passwordIcon, inputType, togglePasswordShow } = usePasswordField();
 
+    whenever(
+      () => appInfo.value?.jwtAuthEnabled as boolean,
+      () => {
+        authenticate({sso: true});
+      },
+      {immediate: true},
+    );
+
     const allowSignup = computed(() => appInfo.value?.allowSignup || false);
 
-    async function authenticate() {
-      if (form.email.length === 0 || form.password.length === 0) {
-        alert.error(i18n.t("user.please-enter-your-email-and-password") as string);
-        return;
+    async function authenticate({sso} = {sso: false}) {
+      if (!sso) {
+        if (form.email.length === 0 || form.password.length === 0) {
+          alert.error(i18n.t("user.please-enter-your-email-and-password") as string);
+          return;
+        }
```

Comment:
For the FE part, I just followed the same approach as #2040 when loading the login screen if appInfo says `jwtAuth` is enabled the form is automatically posted with "SSO", "SSO".
The value there doesn't matter since this won't be ever checked, as the JWT auth code will be executed before.